src/pages/index.js

```diff
@@ -64,68 +77,177 @@ const addCardFormValidator = new FormValidator(validationSettings, addCardForm);
 
 addCardFormValidator.enableValidation();
 
-const addNewCardPopup = new PopupWithForm("#add-card-modal", handleCardSubmit);
+const editAvatarFormValidator = new FormValidator(
+  validationSettings,
+  editAvatarForm
+);
+
+editAvatarFormValidator.enableValidation();
+
+const addNewCardPopup = new PopupWithForm(
+  "#add-card-modal",
+  handleCardSubmit,
+  "Create"
+);
 addNewCardPopup.setEventListners();
 
 const editProfilePopup = new PopupWithForm(
   "#profile-edit-modal",
-  handleProfileEditSubmit
+  handleProfileEditSubmit,
+  "Save"
 );
 editProfilePopup.setEventListners();
 
 const imagePreviewPopup = new PopupWithImage("#preview-image-modal");
 imagePreviewPopup.setEventListners();
 
+const confirmDelete = new PopupWithConfirm({
+  popupSelector: "#confirm-delete-modal",
+  // handler shoould be here
+  handler: (cardId) => {},
+});
+
+const avatarModal = new PopupWithForm(
+  "#avatar-modal",
+  handleUpdateAvatar,
+  "Save"
+);
+avatarModal.setEventListners();
+
+confirmDelete.setEventListners();
+
 const userInfo = new UserInfo({
   userNameSelector: ".profile__title",
   userDescription: ".profile__description",
+  userAvatar: ".profile__image",
 });
 
 function handleImageClick(data) {
   imagePreviewPopup.open(data);
 }
 
+function handleUpdateAvatar({ link }) {
+  avatarModal.renderLoading(true);
+  api
+    .changeProfilePic({ avatar: link })
+    .then((data) => {
+      avatarModal.close();
+      userInfo.setUserInfo(data.name, data.description, data.avatar);
```

Comment:
` data.description`

* When I change the `avatar`, the `job` disappears


`data.about` should be used here


```js
      userInfo.setUserInfo(data.name, data.about, data.avatar);


```